### PR TITLE
Mirroring recent changes to AC model builders, DC model builders now …

### PR DESCRIPTION
**Pull request by jeanpaulwatson**
…require explicit specification of marginal slack penalty costs when slack variables are requested.

### CODE REVIEW

**Comment by bknueven** [comment]
Renaming slack_pos/slack_neg to over_generation/load_shed

**Review by bknueven — APPROVED** [review]
I am happy, but I've now committed at least two major changes -- renaming the slack variables and reporting the slack values in the returned ModelData object -- for this PR.

File: egret/models/acopf.py [hunk 1]
```diff
@@ -620,6 +620,10 @@ def solve_acopf(model_data,
         else:
             b_dict['vm'] = value(m.vm[b])
             b_dict['va'] = value(m.va[b])
+        if hasattr(m, 'p_load_shed'):
```
Comment (jeanpaulwatson) [inline egret/models/acopf.py:623]: Now I get your comment from yesterday regarding things to return, on #162!

**Comment by jeanpaulwatson** [comment]
Enhancements (and typo fixes) look great - merging now...

**Pull request merged**